data/src/main/kotlin/info/dvkr/screenstream/data/httpserver/HttpServerFiles.kt

```diff
@@ -350,6 +351,15 @@ class HttpServerFiles(context: Context, private val settingsReadOnly: SettingsRe
             }
         }
         coplayObject.put("appsList", appsArray)
+
+        val batteryObject = JSONObject()
+        val bm: BatteryManager = this.applicationContext.getSystemService("batterymanager") as BatteryManager
+        batteryObject.put("level", bm.getIntProperty(BatteryManager.BATTERY_PROPERTY_CAPACITY))
+        batteryObject.put("charging", bm.isCharging())
+        coplayObject.put("battery", batteryObject)
+
+        coplayObject.put("serial", android.os.Build.SERIAL)
+
```

Comment:
This is awesome! I went hunting for a way to add the serial number but every method I found was deprecated.

Reply:
Techincally android.os.Build.SERIAL is deprecated, but it works on the Quest as the Quest is running SDK25 (https://stackoverflow.com/a/11029386)